solarforecastarbiter/metrics/preprocessing.py

```diff
@@ -396,7 +396,12 @@ def apply_flag(quality_flag):
             fx_interval_length, closed=closed_obs, label=closed_fx).sum()
         threshold = (
             quality_flag.resample_threshold_percentage / 100. * interval_ratio)
-        flagged = resampled_flags_count > threshold
+        # If threshold is 0, any points being flagged counts, but
+        # don't just throw away all data.
+        if threshold == 0:
+            flagged = resampled_flags_count > threshold
+        else:
+            flagged = resampled_flags_count >= threshold
```

Comment:
@wholmgren What are your thoughts on handling a resample threshold of 0 this way?  If we don't do this, 0 just means "drop all the data".

Reply:
Ohh maybe that is why we chose > originally. I think that would be reasonable. But how would we document and communicate in the user interface?

Reply:
We could either add it to the help text pop up, or add a note below the resample threshold percentage input. I think adding the note below is a better solution, just so that it's visible to users who already understand how these filters work.

Reply:
Sounds good.